lab-isaac/__test__/read-file.test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const fs = require('fs');
+const readFile = require('../lib/read-file');
+
+describe('My FS module', function() {
+  test('should return some data', done => {
+    let first1, second2, third3;
+
+    fs.readFile(`${__dirname}/../assets/one.txt`, (err, data) => {
+      if (err) console.error(err);
+      first1 = data.toString('hex', 0, 8);
+    });
+
+    fs.readFile(`${__dirname}/../assets/two.txt`, (err, data) => {
+      if (err) console.error(err);
+
+      second2 = data.toString('hex', 0, 8);
+    });
+
+    fs.readFile(`${__dirname}/../assets/three.txt`, (err, data) => {
+      if (err) console.error(err);
+
+      third3 = data.toString('hex', 0, 8);
+    });
+
+
+    readFile.read((data) => {
+
+      let odata = [data.first, data.second, data.third];
+      let ndata = [first1, second2, third3];
+      expect(odata).toEqual(ndata);
+      done();
+    });
+  });
+});
+
+test('Testing for improper file path:', done => {
```

Comment:
This test isn't actually running because it is outside your describe block. You would also need to have your expect and done() inside of your callback. Additionally you will need to look at the Jest docs to find an assertion for an error.